src/tests/test_libraries/test_tick.cairo

```diff
@@ -793,62 +799,62 @@ mod TickTests {
 
         #[test]
         #[available_gas(30000000)]
-        #[should_panic(expected: ('int: out of range', 'ENTRYPOINT_FAILED'))]
+        #[should_panic(expected: ('int: out of range',))]
```

Comment:
Why did we make this change, and what is the difference?

Reply:
What do you mean? Isn't necessary to use `ENTRYPOINT_FAILED` for contract functions that panic?